### PR TITLE
Mrshu/more labeler buttons

### PR DESCRIPTION
**Pull request by mrshu**

### CODE REVIEW

File: configapp/static/js/labeler.js [hunk 1]
```diff
@@ -155,3 +155,17 @@ Labeler.prototype.save = function(saveurl) {
         }
     });
 }
+
+Labeler.prototype.set_occupancy_of_all = function(occupancy) {
+    for (var i = 0; i < this.polygons.length; i++) {
+      this.occupies[i] = occupancy;
```
Comment (Adman) [inline configapp/static/js/labeler.js]: 4 spaces here please

File: configapp/static/js/labeler.js [hunk 2]
```diff
@@ -155,3 +155,17 @@ Labeler.prototype.save = function(saveurl) {
...
+    }
+    this.redraw();
+}
+
+Labeler.prototype.reverse_occupancy_of_all = function(occupancy) {
+    for (var i = 0; i < this.polygons.length; i++) {
+      this.occupies[i] = !this.occupies[i];
```
Comment (Adman) [inline configapp/static/js/labeler.js]: also here

**Review by Adman — COMMENTED** [review]
What for are those data images?

**Comment by mrshu** [comment]
@Adman They are already in master if I am not mistaken.

**Pull request merged**